plugins/processors/starlark/starlark_test.go

```diff
@@ -3428,6 +3452,7 @@ def apply(metric):
 			// Ensure we get back the correct number of metrics
 			require.Len(t, acc.GetTelegrafMetrics(), tt.numMetrics)
 			for _, m := range acc.GetTelegrafMetrics() {
+				fmt.Printf("%T\n", m)
 				m.Accept()
```

Comment:
The metric at this point is not a tracking metric, so running accept does not run the delivery function as expected, hence why tests that include the original metric in the result will currently fail as the original metric is never accepted.